### PR TITLE
Fixes lp#1607161: image metadata help updated from Juju 1.x desc.

### PR DESCRIPTION
**Pull request by anastasiamac**
## Description of change

We had a very outdated description of the world, from Juju 1.x...
This PR updated the text and also made a distinction between serving image metadata from local vs from a remote location. 

## Bug reference

https://bugs.launchpad.net/juju/+bug/1607161

### CODE REVIEW

File: cmd/plugins/juju-metadata/imagemetadata.go [hunk 1]
```diff
@@ -176,13 +176,17 @@ Image metadata files have been written to:
...
 image metadata search path. There are 2 options:
 
-1. Use the --metadata-source parameter when bootstrapping:
-   juju bootstrap --metadata-source %s
-
-2. Use image-metadata-url in $JUJU_DATA/environments.yaml
-(if $JUJU_DATA is not set it will try $XDG_DATA_HOME/juju and
-if not set either default to ~/.local/share/juju)
-Configure a http server to serve the contents of
+1. For local access - use the --metadata-source parameter when bootstrapping:
```
Comment (jameinel) [inline cmd/plugins/juju-metadata/imagemetadata.go]: "For local access, "

Not sure what the '-' is doing there.

File: cmd/plugins/juju-metadata/imagemetadata.go [hunk 2]
```diff
@@ -176,13 +176,17 @@ Image metadata files have been written to:
...
+   juju bootstrap [...] --metadata-source %s
```
Comment (jameinel) [inline cmd/plugins/juju-metadata/imagemetadata.go]: Would it be this or
 juju bootstrap --metadata-source %s [...]

File: cmd/plugins/juju-metadata/imagemetadata.go [hunk 3]
```diff
@@ -176,13 +176,17 @@ Image metadata files have been written to:
...
+
+2. For remote access - use image-metadata-url attribute for model configuration.
```
Comment (jameinel) [inline cmd/plugins/juju-metadata/imagemetadata.go]: s/ - /, /

**Comment by anastasiamac** [comment]
$$merge$$

**Pull request merged**